tests/test_dpool.py

```diff
@@ -0,0 +1,120 @@
+"""
+CommandLine:
+    pytest tests/test_dpool.py
+"""
+import torch
+
+from mmdet.ops.dcn.deform_pool import (DeformRoIPoolingPack,
+                                       ModulatedDeformRoIPoolingPack)
+
+
+def test_dpool_empty_cpu():
+    """
+    CommandLine:
+        xdoctest -m tests/test_dpool.py test_dpool_empty_cpu
```

Comment:
No need to use xdoctest to perform unit tests. It is mainly used to test the cases in docstrings.